### DISCUSSION TITLE
How can I disable the Github page of my old account?

### DISCUSSION THREAD

**Discussion by mingolladaniele** [community/community#51697] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi everyone,
last week I changed my telephone number and I lost the access to my old Github account where I had a personal website hosted. Now I've unlinked the email from the previous account and I've created a new one and a new Github page (a clone the of the ones in my old github account).

The issue is that know I've two Github pages, one that is old and another one that will be constantly updated.
How can I resolve this? I remember you that I cannot access to the old github account since the 2FA doesn't work anymore (I cannot receive the SMS with the code).

Thank you in advance and have a good day! :)

**Comment by APolevanov** [comment]
Unfortunately, if you've lost access to your old GitHub account due to 2FA and you don't have any recovery codes, it's unlikely that you'll be able to regain access to the account and delete the old GitHub Pages site yourself. However, you can try reaching out to GitHub Support to see if they can help you.

1. Scroll down to the bottom of the page and click "Contact Support."
2. Click "Account Support."
3. Click "My account has been hacked or I've lost access to it.
4. Fill out the form with the necessary details, including your old GitHub account's username, email, and an explanation of your situation, such as losing access to 2FA.
5. In your request, mention that you'd like to delete the old GitHub Pages site associated with the account, providing the URL of the old site.
6. Submit the form and wait for a response from GitHub Support. They may ask for additional information to verify your identity and prove that you were the owner of the account.

There's no guarantee that GitHub Support will be able to help you regain access to your old account or delete the old GitHub Pages site, but it's worth a try.

**Reply by mingolladaniele** [reply]
I've sent a ticket hoping that it would be enough to resolve this issue. Thanks for your answer! :)
